cmd/mint/update.go

```diff
@@ -9,17 +9,36 @@ var updateCmd = &cobra.Command{
 	Short: "Update versions for base layers and Mint leaves",
 	Use:   "update [flags] [files...]",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		if len(args) > 0 && args[0] == "leaves" {
-			return updateLeaves(args[1:])
+		if len(args) > 0 {
+			switch args[0] {
+			case "base":
+				return updateBase(args[1:])
+			case "leaves":
+				return updateLeaves(args[1:])
+			}
 		}
```

Comment:
Should we error if they pass something other than base and leaves?

Reply:
We allow you specify files here, just like `mint update base [files]` or `mint update leaves [files]`, so if you pass something else that _isn't_ a file, you'll get a file does not exist error.

Reply:
(We could craft a more interesting message, though)

Reply:
Oh! I see, my apologies